src/main/interface/settings.c

```diff
@@ -948,15 +948,13 @@ const clivalue_t valueTable[] = {
 #if defined(USE_TELEMETRY_IBUS)
     { "ibus_sensor",                VAR_UINT8  | MASTER_VALUE | MODE_ARRAY, .config.array.length = IBUS_SENSOR_COUNT, PG_TELEMETRY_CONFIG, offsetof(telemetryConfig_t, flysky_sensors)},
 #endif
-#if defined(USE_TELEMETRY_SMARTPORT)
-    { "smartport_use_extra_sensors", VAR_UINT8 | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_TELEMETRY_CONFIG, offsetof(telemetryConfig_t, smartport_use_extra_sensors)},
-#endif
 #ifdef USE_TELEMETRY_MAVLINK
     // Support for misusing the heading field in MAVlink to indicate mAh drawn for Connex Prosight OSD
     // Set to 10 to show a tenth of your capacity drawn.
     // Set to $size_of_battery to get a percentage of battery used.
     { "mavlink_mah_as_heading_divisor", VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, 30000 }, PG_TELEMETRY_CONFIG, offsetof(telemetryConfig_t, mavlink_mah_as_heading_divisor) },
 #endif
+    { "telemetry_disabled_sensors", VAR_UINT32 | MASTER_VALUE, .config.bitlength = 32, PG_TELEMETRY_CONFIG, offsetof(telemetryConfig_t, disabledSensors)},
```

Comment:
Attempted to add support for var_uint32 to direct mode.

Reply:
This should use `MODE_BITSET`.

Reply:
I thought MODE_BITSET was to set an individual flag in a bit mask, not the whole mask: (mask = 1 << var->config.bitpos in cli.c)

Reply:
Good point, setting the bitmask for all sensors at once here makes more sense. But can you please split out support for `VAR_UINT32` into a separate pull request so this can be discussed / approved first?

Also, instead of `.config.bitlength = 32`, this should rather be the effective number of sensors.

Reply:
I'll do that, and also one for USE_ESC_SENSOR_TELEMETRY so F3 space can be freed up sooner.